src/tuner/nccl_ofi_regions.cpp

```diff
@@ -10,8 +10,18 @@
 
 #include "tuner/nccl_ofi_tuner_region.h"
 #include "nccl_ofi_param.h"
+#include "nccl_ofi_system.h"
 
 
+static const int NCCL_LL128_LINESIZE = 128;
+static const int NCCL_LL128_LINEELEMS = 16;
```

Comment:
From my limited context I personally don't understand what these variables represent and why we need to use them when tuning the number of channels. Would it be appropriate to add comments describing the variables, as well as why the particular values are chosen? (e.g. do they represent NCCL default values for a particular NCCL version, did we empirically determine values, etc).